templates/pkg/crd_sdk.go.tpl

```diff
@@ -26,9 +26,9 @@ import (
 
 // Hack to avoid import errors during build...
 var (
-    _ = &metav1.Time{}
-    _ = &svcsdk.{{ .SDKAPIInterfaceTypeName}}{}
-    _ = &svcapitypes.{{ .CRD.Names.Camel }}{}
+	_ = &metav1.Time{}
+	_ = &svcsdk.{{ .SDKAPIInterfaceTypeName}}{}
+	_ = &svcapitypes.{{ .CRD.Names.Camel }}{}
```

Comment:
This fixed up some spaces that I'd accidentally put in the template instead of TABs.